requirements/app/app.txt

```diff
@@ -1,6 +1,6 @@
-lightning-cloud ==0.5.46  # Must be pinned to ensure compatibility
+lightning-cloud == 0.5.48  # Must be pinned to ensure compatibility
 packaging
-typing-extensions >=4.0.0, <4.8.0
+typing-extensions >=4.4.0, <4.8.0
```

Comment:
why here, this was another PR

Reply:
CI was breaking last night. This seemed this was required.